examples/tests/design.test.ts

```diff
@@ -0,0 +1,36 @@
+import { shortest } from "@antiwork/shortest";
+
+// Material Design 3 color system
```

Comment:
Remove all comments unless it's about tech debt or some bug or something that isn't obvious from the code itself